src/Frame/Frame.cc

```diff
@@ -1119,8 +1123,18 @@ bool Frame::FillSpatialProfileData(PointXy point, std::vector<CARTA::SetSpatialR
         } else {
             casacore::Slicer section = GetImageSlicer(AxisRange(x), AxisRange(y), AxisRange(CurrentZ()), stokes);
             const auto N = section.length().product();
+#if defined(__APPLE__) && defined(__MACH__) && defined(__MAC_OS_X_VERSION_MIN_REQUIRED)
```

Comment:
There must be a neater way to do this 🤣

Reply:
Would it not be better to add `&& __MAC_OS_X_VERSION_MIN_REQUIRED < 110000` to the outer conditional instead of having a nested conditional with two identical `else` bodies?